### PR TITLE
make height adjustment

### LINKED ISSUE #5397: Height of RemixAI UI is not Responsive
The RemixAI ui is overheight so it can't close this ui.

image

### PR DESCRIPTION
**Pull request by joeizang**
fixes #5397

### CODE REVIEW

**Review by pr15il — CHANGES_REQUESTED** [review]
I think `maxHeight` should be kept at `40rem` and `height` changed to `67%`

File: apps/remix-ide/src/app/components/popup-panel.tsx [hunk 1]
```diff
@@ -89,10 +89,10 @@ export class PopupPanel extends AbstractPanel {
       <div
         className={`px-0 bg-light border-info ${appState?.showPopupPanel ? 'd-flex' : 'd-none'}`}
         style={{
-          maxHeight: '40rem',
+          maxHeight: '60%',
```
Comment (pr15il) [inline apps/remix-ide/src/app/components/popup-panel.tsx:92]:
```suggestion
          maxHeight: '40rem',
```

File: apps/remix-ide/src/app/components/popup-panel.tsx [hunk 2]
```diff
@@ -89,10 +89,10 @@ export class PopupPanel extends AbstractPanel {
...
           maxWidth: '25rem',
           width: 'max-content',
-          height: '40rem',
+          height: '60%',
```
Comment (pr15il) [inline apps/remix-ide/src/app/components/popup-panel.tsx:95]:
```suggestion
          height: '67%',
```

**Review by pr15il — CHANGES_REQUESTED** [review]

File: apps/remix-ide/src/app/components/popup-panel.tsx (hunk 1 above)
Comment (LianaHus) [reply apps/remix-ide/src/app/components/popup-panel.tsx:92]: I think we can even remove max-height

File: apps/remix-ide/src/app/components/popup-panel.tsx (hunk 2 above)
Comment (joeizang) [reply apps/remix-ide/src/app/components/popup-panel.tsx:95]: Ok will have a look

**Review by Aniket-Engg — APPROVED** [review]
height is responsive now

**Comment by pr15il** [comment]
When will it be released if I may know?

**Pull request merged**